doc/user/content/cloud/get-started-with-cloud.md

```diff
@@ -13,23 +16,35 @@ This guide walks you through getting started with Materialize Cloud, from settin
 
 * Signing up for Materialize Cloud
 
-* Creating, and connecting to a Materialize Cloud deployment
+* Creating and connecting to a Materialize Cloud deployment
 
 * Connecting to a streaming data source
 
 * Creating a materialized view
 
 * Exploring common patterns like joins and time-windowing
 
-## Sign up, deploy, connect
+## Sign up and deploy
 
-1. Sign up for Materialize Cloud at [https://cloud.materialize.com](https://cloud.materialize.com#signup).
+1. Sign up for Materialize Cloud at [https://cloud.materialize.com](https://cloud.materialize.com/signup/).
 
-1. Once you [log in](https://cloud.materialize.com), you can use the [Deployments](https://cloud.materialize.com/deployments) page to create, upgrade, or destroy deployments, as well as to obtain the TLS certificates you need to install on your local machine to connect.
+1. Once your account has been created, [log in](https://cloud.materialize.com).
 
-   By default, you can create up to two deployments. If you're interested in more, [let us know](../support)!
+1. If you've been invited to an existing workspace, you can go directly to the [connection instructions](#connect). Otherwise, a dialog asks you to create a workspace. Enter a name and click **Next**.
 
-1. On the [Deployments](https://cloud.materialize.com/deployments) page, click **Create deployment**. Materialize creates a deployment and assigns it a `name` and `hostname`.
+1. Materialize Cloud asks if you want to invite other users. For the purposes of this demonstration, just click **Skip**. (You can always add other users later.)
```

Comment:
This bit doesn't presently exist.

Reply:
Ah, I guess this covers my previous comment.